alarms/FacebookPages/facebookpages_alarm.py

```diff
@@ -0,0 +1,86 @@
+#Setup logging
+import logging
+log = logging.getLogger(__name__)
+
+#Python Modules
+from datetime import datetime
+
+#Local Modules
+from ..alarm import Alarm
+from ..utils import *
+
+#Exernal Modules
+import facebook
+
+class FacebookPages_Alarm(Alarm):
+
+	_defaults = {
+		'pokemon':{
+			'message': "A wild <pkmn> has appeared! Available until <24h_time> (<time_left>).",
+			'location': "True"
+		},
+		'pokestop':{
+			'message': "Someone has placed a lure on a Pokestop! Lure will expire at <24h_time> (<time_left>).",
+			'location': "True"
+		},
+		'gym':{
+			'message':"A Team <old_team> gym has fallen! It is now controlled by <new_team>.",
+			'location': "True"
+		}
+	}
+
+	#Gather settings and create alarm
+	def __init__(self, settings):
+		#Service Info
+		self.page_access_token = settings['page_access_token']
+		self.startup_message = settings.get('startup_message', "True")
+		self.startup_list = settings.get('startup_list', "True")
+				
+		#Set Alerts
+		self.pokemon = self.set_alert(settings.get('pokemon', {}), self._defaults['pokemon'])
+		self.pokestop = self.set_alert(settings.get('pokestop', {}), self._defaults['pokestop'])
+		self.gym = self.set_alert(settings.get('gyms', {}), self._defaults['gym'])
+		
+		#Connect and send startup messages
+		self.connect()
+		timestamps = get_timestamps(datetime.utcnow());
+		if parse_boolean(self.startup_message):
+			self.client.put_wall_post(message="%s - PokeAlarm has intialized!" % timestamps[2])
+		log.info("FacebookPages Alarm intialized.")
+		
+	#Establish connection with FacebookPages
+	def connect(self):
+		self.client = facebook.GraphAPI(self.page_access_token)
+			
+	#Set the appropriate settings for each alert
+	def set_alert(self, settings, default):
+		alert = {}
+		alert['message'] = settings.get('message', default['message'])
+		
+		if parse_boolean(settings.get('location', default['location'])):
```

Comment:
Perhaps you should add the field 'location' to a customizable field called link instead so users can change the link if needed (if they want to customize it to their map or anything like that)